modules/nf-core/dragmap/align/dragmap-align.diff

```diff
@@ -0,0 +1,30 @@
+Changes in module 'nf-core/dragmap/align'
+--- modules/nf-core/dragmap/align/main.nf
++++ modules/nf-core/dragmap/align/main.nf
+@@ -4,8 +4,8 @@
+ 
+     conda "${moduleDir}/environment.yml"
+     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+-        'https://depot.galaxyproject.org/singularity/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:7eed251370ac7f3537c3d9472cdb2f9f5d8da1c5-0':
+-        'biocontainers/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:7eed251370ac7f3537c3d9472cdb2f9f5d8da1c5-0' }"
++        'https://depot.galaxyproject.org/singularity/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:5ebebbc128cd624282eaa37d2c7fe01505a91a69-0':
++        'biocontainers/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:5ebebbc128cd624282eaa37d2c7fe01505a91a69-0' }"
+ 
+     input:
+     tuple val(meta) , path(reads)
+
+--- modules/nf-core/dragmap/align/environment.yml
++++ modules/nf-core/dragmap/align/environment.yml
+@@ -4,7 +4,7 @@
+   - bioconda
+   - defaults
+ dependencies:
+-  - dragmap=1.3.0
++  - dragmap=1.2.1
```

Comment:
I think we don't need to patch anymore and it should work now with the latest version. The rebuild conda recipe fixed it.

Reply:
ok, I'll do that in a separate PR then, as this is out of scope here